test/test_lyrics.py

```diff
@@ -457,28 +432,34 @@ def setUp(self):
             self.skipTest("Python's built-in HTML parser is not good enough")
 
 
-class LyricsGeniusScrapeTest(LyricsGeniusBaseTest):
-
-    """Checks that Genius backend works as intended.
-    """
-    import requests
+class GeniusScrapeLyricsFromSongPageTest(GeniusBaseTest):
+    """tests Genius.scrape_lyrics_from_song_page()"""
 
     def setUp(self):
         """Set up configuration"""
-        LyricsGeniusBaseTest.setUp(self)
+        GeniusBaseTest.setUp(self)
         self.plugin = lyrics.LyricsPlugin()
 
-    @patch.object(requests, 'get', GeniusMockGet())
     def test_no_lyrics_div(self):
         """Ensure that `lyrics_from_song_page` doesn't crash when the html
         for a Genius page doesn't contain <div class="lyrics"></div>
         """
         # https://github.com/beetbox/beets/issues/3535
         # expected return value None
-        song_url = 'https://genius.com/sample'
-        self.assertEqual(genius.lyrics_from_song_page(song_url),
-                         None)
+        url = 'https://genius.com/sample'
+        mock = MockFetchUrl()
+        self.assertEqual(genius.scrape_lyrics_from_song_page(mock(url)), None)
+
+    def test_good_lyrics(self):
+        """Ensure we are able to scrape a page with lyrics"""
+        url = 'https://genius.com/Wu-tang-clan-cream-lyrics'
+        mock = MockFetchUrl()
+        self.assertIsNotNone(genius.scrape_lyrics_from_song_page(mock(url)))
+
+    # TODO: find an example of a lyrics page with multiple divs and test it
```

Comment:
I plan on leaving this one out as a `byte-sized` or perhaps `first-timers-only` issue ;)